modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -87,8 +87,8 @@ $ oc get route -n openshift-hw-events
 .Example output
 [source,terminal]
 ----
-NAME                      HOST/PORT                                                                            PATH   SERVICES                  PORT    TERMINATION            WILDCARD
-hw-event-proxy            hw-event-proxy-cloud-native-events.apps.compute-1.example.com                               hw-event-proxy-service    9087    edge                   None
+NAME             HOST/PORT                                                       PATH   SERVICES                 PORT   TERMINATION   WILDCARD
+hw-event-proxy   hw-event-proxy-openshift-hw-events.apps.compute-1.example.com          hw-event-proxy-service   9087   edge          None
```

Comment:
I'm not sure about the apps name, as far as I know, this name would not change. @nishant-parekh @imiller0 could you please help confirm on this name?

Same for apps name in rest of the doc.

Reply:
So `hw-event-proxy-openshift-hw-events.apps.compute-1.example.com` is correct? @nishant-parekh

Reply:
this should be : `hw-event-proxy-cloud-native-events.apps.compute-1.example.com`